### PR TITLE
feat: update deployment on mainnet

### PR DESCRIPTION
**Pull request by cosinlink**
### Description
update deployment on mainnet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,25 @@ npm run deploy:bsc
 
 ## Contract Entrypoint
```
Comment (zjubfd) [inline README.md:124]: Compare to `### Greenfield Contracts on BSC Testnet` chapter, can you add a new chapter like `### Greenfield Contracts on BSC Mainnet`

File: README.md [hunk 2]
```diff
@@ -122,6 +122,25 @@ npm run deploy:bsc
...
+| GroupHub      | 0xDd9af4573D64324125fCa5Ce13407be79331B7F7 |
+
+Extra:
+
+| contract name | address                                    |
+|---------------|--------------------------------------------|
+| Deployer      | 0x4763c12b21a548BCbD22a682fb15930565e27C43 |
+| ProxyAdmin    | 0xf9010DC773eE3961418C96dc67Fc5DcCB3EA2C08 |
+| LightClient   | 0x433bB48Bd86c089375e53b2E2873A9C4bC0e986B |
+| RelayerHub    | 0x31C477F05CE58bB81A9FB4b8c00560f1cBe185d1 |
```
Comment (zjubfd) [inline README.md:142]: can we have more detailed contracts? we get:
```
  "BlockNumber": 32332785,
  "EmergencyOperator": "0xbC3380aA2B1f0a7aD4c58b410D6Ec98bFb1e12f1",
  "EmergencyUpgradeOperator": "0xfBC18196967bEbF94513a85F770BF7b62d97c80E",
  "Deployer": "0x4763c12b21a548BCbD22a682fb15930565e27C43",
  "ProxyAdmin": "0xf9010DC773eE3961418C96dc67Fc5DcCB3EA2C08",
  "GovHub": "0x1c9766EbcA1f38A06A04947129B394bF7FEc4599",
  "CrossChain": "0x77e719b714be09F70D484AB81F70D02B0E182f7d",
  "TokenHub": "0xeA97dF87E6c7F68C9f95A69dA79E19B834823F25",
  "LightClient": "0x433bB48Bd86c089375e53b2E2873A9C4bC0e986B",
  "RelayerHub": "0x31C477F05CE58bB81A9FB4b8c00560f1cBe185d1",
  "BucketHub": "0xE909754263572F71bc6aFAc837646A93f5818573",
  "ObjectHub": "0x634eB9c438b8378bbdd8D0e10970Ec88db0b4d0f",
  "GroupHub": "0xDd9af4573D64324125fCa5Ce13407be79331B7F7",
  "AdditionalBucketHub": "0x7B09204a5F2852aec536404777F91B123020b1C8",
  "AdditionalObjectHub": "0xaE8cd47A456d8b750aE4e0c73A1D7B245aC0D20F",
  "AdditionalGroupHub": "0x8b93001e9fc20c92305A3a4Eb610302569b26Ab5",
  "BucketERC721Token": "0xC92d1d4b64Aebe28042206656b9E70E571A0E5eF",
  "ObjectERC721Token": "0x4B92705a60d69f7A96aaDB8faa892526eB71adb7",
  "GroupERC721Token": "0x943FAC6CEBE6e45CE59bA911E5B6447c1a991450",
  "MemberERC1155Token": "0xAb73f243Be4d0fC5644c822351eC77e85DC2B5Ea",

```

File: README.md (hunk 1 above)
Comment (cosinlink) [reply README.md:136]: added

File: README.md (hunk 2 above)
Comment (cosinlink) [reply README.md:154]: added `for full list of contracts, please refer to:`

**Review by zjubfd — APPROVED** [review]

**Pull request merged**